cockroachcloud/serverless-faqs.md

```diff
@@ -0,0 +1,95 @@
+---
+title: CockroachCloud Serverless (beta) FAQs
+summary: Get answers to frequently asked questions about CockroachCloud Serverless (beta)
+toc: true
+---
+
+This page answers the frequently asked questions about CockroachCloud Serverless (beta) and CockroachCloud Dedicated.
+
+<div class="filters clearfix">
+    <a href="serverless-faqs.html"><button class="filter-button page-level current">CockroachCloud Serverless</button></a>
+    <a href="frequently-asked-questions.html"><button class="filter-button page-level">CockroachCloud Dedicated</button></a>
+</div>
+
+## General
+
+### What is CockroachCloud Serverless (beta)?
+
+CockroachCloud Serverless (beta) delivers free CockroachDB clusters for you and your organization. It is a managed instance of CockroachDB that removes the friction of initial cluster sizing and auto-scales based on your application traffic.
+
+### How do I start using CockroachCloud Serverless (beta)?
+
+To get started with CockroachCloud Serverless (beta), <a href="https://cockroachlabs.cloud/signup?referralId=docs_serverless_faq" rel="noopener" target="_blank">sign up for a CockroachCloud account</a>, click **Create Cluster**, then click **Create your free cluster**. Your cluster will be ready in 20-30 seconds. For more information, see [**Quickstart**](quickstart.html).
+
+### What are the usage limits of Cockroach Cloud Serverless (beta)?
+
+Free clusters have a limit of 100M Request Units per month and 5GB of storage, but the performance is throttled to a maximum of 100 RUs/second. Paid clusters have access to the same resources with no limitations in addition to the amount you pay for.
+
+### What is a Request Unit?
+
+All resource usage in CockroachCloud Serverless (beta) is measured in Request Units, or RUs. RUs represent the compute and I/O resources used by a read or a write query. All database operations cost a certain amount of RUs depending on the resources used. For example, a "small read" might cost 1 RU, and a "large read" such as a full table scan with indexes could cost 100 RUs.
+
+### Do I have to pay for CockroachCloud Serverless (beta)?
+
+No, you can create a Serverless cluster that is free forever. If you choose to create a paid cluster, you will only be charged for the resources you use up to your spend limit.
```

Comment:
I think the term "choose to create a paid cluster" could be confusing because it implies that being free or paid is an immutable property of the cluster itself, but in reality the budget of a cluster can be set to zero or a positive number after the cluster has been created.